index.html

```diff
@@ -1,28 +1,101 @@
+                <td class="td-b" colspan="4">&nbsp;</td>
+            </tr>
+
+            <!-- ======= regions ======= -->
+            <tr id="AL">
+                <td class="td-r"><div class="region"><p class="label-r">AL</p></div></td>
+                <td class="td-b"><div class="buffer ft500">&nbsp;</div></td>
+                <td class="td-b"><div class="buffer ft1000">&nbsp;</div></td>
+                <td class="td-b"><div class="buffer ft2500">&nbsp;</div></td>
+                <td class="td-b"><div class="buffer ft5280">&nbsp;</div></td>
+            </tr>
+            <tr id="AR">
+                <td class="td-r"><div class="region"><p class="label-r">AR</p></div></td>
+                <td class="td-b"><div class="buffer ft500">&nbsp;</div></td>
+                <td class="td-b"><div class="buffer ft1000">&nbsp;</div></td>
+                <td class="td-b"><div class="buffer ft2500">&nbsp;</div></td>
+                <td class="td-b"><div class="buffer ft5280">&nbsp;</div></td>
+            </tr>
+            <tr id="DC">
+                <td class="td-r"><div class="region"><p class="label-r">DC</p></div></td>
+                <td class="td-b"><div class="buffer ft500">&nbsp;</div></td>
+                <td class="td-b"><div class="buffer ft1000">&nbsp;</div></td>
+                <td class="td-b"><div class="buffer ft2500">&nbsp;</div></td>
+                <td class="td-b"><div class="buffer ft5280">&nbsp;</div></td>
+            </tr>
+            <tr id="MO">
+                <td class="td-r"><div class="region"><p class="label-r">MO</p></div></td>
+                <td class="td-b"><div class="buffer ft500">&nbsp;</div></td>
+                <td class="td-b"><div class="buffer ft1000">&nbsp;</div></td>
+                <td class="td-b"><div class="buffer ft2500">&nbsp;</div></td>
+                <td class="td-b"><div class="buffer ft5280">&nbsp;</div></td>
+            </tr>
+            <tr id="PG">
+                <td class="td-r"><div class="region"><p class="label-r">PG</p></div></td>
+                <td class="td-b"><div class="buffer ft500">&nbsp;</div></td>
+                <td class="td-b"><div class="buffer ft1000">&nbsp;</div></td>
+                <td class="td-b"><div class="buffer ft2500">&nbsp;</div></td>
+                <td class="td-b"><div class="buffer ft5280">&nbsp;</div></td>
+            </tr>
+        </table>
+        <div class="locText start">&nbsp;</div><span class="filter-label">start</span>
+        <div class="locText end">&nbsp;</div><span class="filter-label">end</span>
+    </div>
+
+    <!-- ======= map ======= -->
+    <div id="map"></div>
+
+    <script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
+    <script src="https://api.tiles.mapbox.com/mapbox.js/v2.4.0/mapbox.js"></script>
+    <script src='script.js'></script>
+
+    <script type="text/javascript">
```

Comment:
Removed 94-95 (and same for other 4 regions).  Kept laneFiles object (3 current laneType files  for DC) to preserve consistent structure and for future lane type distinguishment for other regions